src/components/Overview/PlotlyCharts/DataCollectedChart.tsx

```diff
@@ -7,58 +6,78 @@ import {
   AvastarParsedDataPointState,
 } from '../../../types/dataTypes';
 
+
 const Plot = createPlotlyComponent(Plotly);
+interface PropsFilter {
+  platform: string;
+  origin: string;
+  type: string;
+}
```

Comment:
Comme mentionné pour le fichier Filter.tsx, migrer l'interface vers `dataTypes.ts` ?

Ensuite, tu peux juste faire `import { PropsFilter } from '../types/dataTypes'`;

Reply:
I have ts errors when I try to do it.... if you have the solution please commit it

Reply:
Done =)